### ISSUE TITLE
find_element returns stalled data

### ISSUE THREAD

**Issue by lukaslueg** [atroche/rust-headless-chrome#92]
[This test](https://github.com/lukaslueg/macro_railroad_ext/blob/34b2d38078a18536f5a8faf2c8b176c980a0028c/extension/integration_test/src/main.rs#L126) causes an (SVG) element to be removed from the DOM via page interactions and checks if the `Element` is actually gone. This succeeds locally but fails on Travis for unknown reasons. A debug-heavy log is found [here](https://api.travis-ci.org/v3/job/504278181/log.txt).

I don't know the protocol-part at all, yet my feeling is that we hold on to the remote object via `let _ = tab.find_element(LEGEND)?;`, causing it to stay valid. Is that a possibility?

**Comment by lukaslueg** [comment]
Some spooky action at a distance: The test fails locally as well if run under xvfb. I took a screenshot and noticed that the DOM-Node I'm asking for is outside the 640x480ish window xvfb creates. After I increased the virtual screen size, the test succeeds...

I have no idea why a DOM-Node refuses to disappear if drawn outside the viewport of a given screen resolution. Yet apparently this is a thing.

In any case, [`macro_railroad_ext`](https://github.com/lukaslueg/macro_railroad_ext) now has a working CI.
